examples/summarization/README.md

```diff
@@ -9,19 +12,47 @@ tar -xzvf cnn_dm.tgz
 this should make a directory called cnn_dm/ with files like `test.source`. 
 To use your own data, copy that files format. Each article to be summarized is on its own line.
 
+#### For T5 models
+First, you need to download the CNN data. It's about ~400 MB and can be downloaded by 
+running 
+
+```bash
+python download_cnn_daily_mail.py cnn_articles_input_data.txt cnn_articles_reference_summaries.txt
+```
+
+You should confirm that each file has 11490 lines:
+
+```bash
+wc -l cnn_articles_input_data.txt # should print 11490
+wc -l cnn_articles_reference_summaries.txt # should print 11490
+```
+
 ### Evaluation
+
+#### For BART models
 To create summaries for each article in dataset, run:
 ```bash
 python evaluate_cnn.py <path_to_test.source> cnn_test_summaries.txt
 ```
 the default batch size, 8, fits in 16GB GPU memory, but may need to be adjusted to fit your system.
 
+#### For T5 models
+To create summaries for each article in dataset, run:
+```bash
```

Comment:
This command seems wrong since it doesn't mention t5 at all.
I also think there should be one command:

```bash
python evaluate_cnn.py <path_to_test.source> test_generations.txt <model_name> rouge_scores.txt
```